index.html

```diff
@@ -0,0 +1,176 @@
+        </div>
+      </section>
+      <section class="program" id="program">
+        <h3 class="program-title">Main Program</h3>
+        <div class="program-container">
+          <div class="program-card">
+            <div class="left">
+              <img src="./images/motivation.png" alt="motivation-icone" />
+              <span>Motivation</span>
+            </div>
+            <div class="right">
+              Listen to the motivation form various personalities about the
+              message of sharing and opening opportunities to the less
+              privileged in your community.
+            </div>
+          </div>
+          <div class="program-card">
+            <div class="left">
+              <img src="./images/forum.png" alt="forum-icone" />
+              <span>Forum</span>
+            </div>
+            <div class="right">
+              Have the time to share your toughts and opinions with our team for
+              each topic.
+            </div>
+          </div>
+          <div class="program-card">
+            <div class="left">
+              <img src="./images/lecture.png" alt="lecture-icon" />
+              <span>Donation</span>
+            </div>
+            <div class="right">
+              The value of life is not in its duration, but in its donation.
+              What a privilege to be here on the planet to contribute your
+              unique donation to humankind!
+            </div>
+          </div>
+          <div class="program-card">
+            <div class="left">
+              <img src="./images/support.png" alt="support-icone" />
+              <span>Support</span>
+            </div>
+            <div class="right">
+              Whatever little it maybe, it means a lot to someone out there.
+              Support the needy in our community.
+            </div>
+          </div>
+          <div class="program-card">
+            <div class="left">
+              <img src="./images/workShop.png" alt="workshop-icone" />
+              <span>Workshop</span>
+            </div>
+            <div class="right">
+              Get oppotunities to network with GC affiliates around the globe.
+            </div>
+          </div>
+        </div>
+        <div class="join-btn">
+          <a href="#">SEE THE WHOLE PROGRAM</a>
+        </div>
+      </section>
+      <section id="sponsor" class="sponsor"></section>
+      <section class="partner">
+        <h3 class="partner-title">Partner</h3>
+        <div class="partner-container">
+          <figure>
+            <img src="./images/mozilla.png" alt="img1" />
+          </figure>
+          <figure>
+            <img src="./images/google.png" alt="img2" />
+          </figure>
+          <figure>
+            <img src="./images/naver.png" alt="img3" />
+          </figure>
+          <figure>
+            <img src="./images/airbnb.jpg" alt="img4" />
+          </figure>
+        </div>
+      </section>
```

Comment:
- [ ] Kindly try to match the Behance design in the `partners` section on the home page mobile version by increasing the `image size` of each brand, increasing top spacing, and decreasing button spacing.

| Your design | Behance design |
| --- | --- | 
| ![image](https://user-images.githubusercontent.com/24703171/164565133-841dfa58-06fa-4d53-92ef-1c79b7f0ffb5.png) | ![image](https://user-images.githubusercontent.com/24703171/164565160-c593a440-8ce9-4813-b2be-90d84d992d07.png) |

- [ ] and also the same in the desktop version

| Your design | Behance design |
| --- | --- | 
| ![image](https://user-images.githubusercontent.com/24703171/164565352-8b3a6a92-c090-4292-a393-ac684a47cdd3.png) | ![image](https://user-images.githubusercontent.com/24703171/164565309-2c92e474-bbf0-4db7-8812-88482c60b86c.png) |